.github/workflows/fetch_repos.yml

```diff
@@ -0,0 +1,38 @@
+name: Fetch Repos
+
+on:
+  schedule:
+    - cron: '0 0 */7 * *' 
+
+env:
+  GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+jobs:
+  inspect-model:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Setup Conda Environment
```

Comment:
Can we use the conda-incubator action to setup conda? Like here: https://github.com/ersilia-os/ersilia/blob/0685109cbe68ebb3950301e778020c3c6cb87677/.github/workflows/tests_and_cleanup.yml#L64?